src/PresentationalComponents/NumberDescription/NumberDescription.js

```diff
@@ -3,14 +3,14 @@ import { Button } from '@patternfly/react-core/dist/js/components/Button/Button'
 import propTypes from 'prop-types';
 import '../../../src/PresentationalComponents/NumberDescription/NumberDescription.scss';
 import { NumberData } from '../../PresentationalComponents/NumberData/NumberData';
-import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import CriticalIcon from '../../Icons/CriticalIcon';
 
-export const NumberDescription = ({ data, dataSize, link, linkDescription, layout, critical }) => (
-    <div className={ `ins-c-dashboard__number-description pf-m-${layout}` }>
+export const NumberDescription = ({ data, dataSize, percentageData, link, linkDescription, layout, critical }) => (
+    <div className={ `ins-c-dashboard__number-description` + (layout ? ` pf-m-${layout}` : '') }>
```

Comment:
What are your thoughts on using string literals instead of concatenation? You are already using the string literal, might as well remove the concaenation and fully use it.

Reply:
Good point, updated in a few places